plugins/collections/README.md

```diff
@@ -53,3 +53,23 @@ Example payload:
     "pipelineIds": [12, 35, 47, 89]
 }
 ```
+
+#### Update a collection
+You can update the name, description, or pipelineIds of a collection.
+
+`PUT /collections/{id}`
+
+**Arguments**
+
+* `name` - An optional new name for the collection. Names must be unique.
+* `description` - An optional new description for the collection.
+* `pipelineIds` - An optional new array of ids of pipelines to be added to the collection.
```

Comment:
So is there/will there be any API route that just adds pipelines to a collection? Or would the workflow have to be for someone to check which pipelines are already in the collection and then send them along with the new ones they want to add to this route?